Euphorolog/Filter/PaginationFilter.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Euphorolog.Filter
+{
+    public class PaginationFilter
+    {
+        public int pageNumber { get; set; }
+        public int pageSize { get; set; }
+        public PaginationFilter()
+        {
+            this.pageNumber = 1;
+            this.pageSize = 10;
+        }
+        public PaginationFilter(int pageNumber, int pageSize)
+        {
+            this.pageNumber = pageNumber < 1 ? 1 : pageNumber;
+            this.pageSize = pageSize > 10 ? 10 : pageSize;
```

Comment:
Use private variables such as `_maxPageSize = 10` or `_startPage = 1` and use them in expressions. So that people have some idea on where these number 10 came from.